app/controllers/application_controller.rb

```diff
@@ -17,4 +17,17 @@ def init_graphql_client
     @smart_village = SmartVillageApi.new(user: @current_user).client
   end
 
+  def load_category_list
+    results = @smart_village.query <<~GRAPHQL
+    query {
+      categories() {
```

Comment:
die `()` sind nicht nötig, oder? haben mich auf anhieb verwirrt. da keine parameter übergeben werden, können die weg?

Reply:
vllt einrücken zwischen <<~GRAPHQL und GRAPHQL? weiß nicht wie da best practice ist.

Reply:
stimmt können weg... in Ruby hätte ich das auch gleich erkannt, aber mit der graphql Syntax bin ich **noch** nicht so fit

Reply:
Einrückung habe ich noch um eines erhöht... codeclimate hatte das auch angemerkt